packages/core/src/providers/netsuite.ts

```diff
@@ -1,243 +1,225 @@
+ *     }
+ *   };
+ *
+ *   return {
+ *     get,
+ *   };
+ * );
+ * ```
+ *
+ * > **Note**: Above is an example of returning the basic runtime information. Be sure to create a new script record and deployment record. Upon saving the deployment record. We will get our URLs for our RESTlet.
+ *
+ * ## Configuration
+ *
+ * ```ts
+ * import { Auth } from "@auth/core"
+ * import Netsuite from "@auth/core/providers/netsuite"
+ *
+ * const request = new Request(origin)
+ * const response = await Auth(request, {
+ *   providers: [
+ *       NetSuite({
+ *         accountID: NETSUITE_ACCOUNT_ID, // EX: TSTDRV1234567 or 81555 for prod, and 1234567-SB1 for Sandbox accounts not "_" use "-".
+ *        // Returns the current user using the N/runtime module. This url can be a suitelet or RESTlet (Recommended)
+ *        // Using getCurrentUser(); So we match this schema returned from this RESTlet in the profile callback. (Required)
+ *         userinfo: "https://1234567.restlets.api.netsuite.com/app/site/hosting/restlet.nl?script=123&deploy=1",
+ *       })
+ *   ],
+ * })
+ * ```
+ *
+ * ### Resources
+ *
+ * - [NetSuite - Creating an Integration Record (OAuth 2.0)](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_157771733782.html#Related-Topics)
+ * - [NetSuite - Authorizing OAuth Requests](https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps)
+ * - [NetSuite - Configure OAuth Roles](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_157771510070.html#Set-Up-OAuth-2.0-Roles)
+ * - [Learn more about NetSuite OAuth 2.0](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/chapter_157769826287.html#OAuth-2.0)
+ *
+ * ### Notes
+ *
+ * :::tip
+ * Make sure the `userinfo` matches the return type of the profile callback to ensure the user session gets read correctly.
+ * To override the defaults for your use case, check out [customizing a built-in OAuth provider](https://authjs.dev/guides/providers/custom-provider#override-default-options).
+ * :::
+ *
+ */
+export default function NetSuite<P extends NetSuiteProfile>(
+  config: OAuthUserConfig<P> & OAuthNetSuiteOptions
+): OAuthConfig<P> {
+  const { accountID } = config
+
+  return {
+    id: "netsuite",
+    name: "NetSuite",
+    type: "oauth",
+    checks: ["state"],
+    authorization: {
+      url: `https://${accountID}.app.netsuite.com/app/login/oauth2/authorize.nl`,
+      params: { scope: "restlets rest_webservices" },
+    },
+    token: `https://${accountID}.suitetalk.api.netsuite.com/services/rest/auth/oauth2/v1/token`,
+    profile(profile) {
+      // This is the default runtime.getCurrentUser() object returned from the RESTlet or SUITELet
+      return {
+        id: profile.id.toString(),
+        name: profile.name,
+        email: profile.email,
+        image: null,
+      }
+    },
+    style: { logo: "/netsuite.svg", bg: "#3a4f5f", text: "#fff" },
```

Comment:
This needs to be swapped out. 

Can you update that to be this color for the background #3a4f5f ➡️ #181a1b ?